invokeai/frontend/web/src/app/components/ImageDnd/typesafeDnd.tsx

```diff
@@ -78,14 +82,12 @@ export type ImageDraggableData = BaseDragData & {
   payload: { imageDTO: ImageDTO };
 };
 
-export type ImageNamesDraggableData = BaseDragData & {
-  payloadType: 'IMAGE_NAMES';
-  payload: { image_names: string[] };
+export type ImageDTOsDraggableData = BaseDragData & {
+  payloadType: 'IMAGE_DTOS';
+  payload: { imageDTOs: ImageDTO[] };
```

Comment:
I changed the drag and drop payload for multiple images to be an array of DTOs instead of just image names. This gives us more flexibility when handling these dnd events. 

For example, we now have access to an image's current `board_id` when dropping multiple images on a board. This allows us to greatly reduce the number of network requests needed when doing batched board management operations.